README.md

```diff
@@ -244,6 +244,11 @@ app.use(async function verifications(req, res, next){
   - paystack.fetchTransfer()
   - paystack.finalizeTransfer()
   - paystack.initiateBulkTransfer()
+- transfer_recipients
+  - paystack.createTransferRecipient
```

Comment:
Each line from line 248 to 251 should be formatted with open and close parenthesis to be in unison to the rest of every other section